### PR TITLE
dev-DPRO-2600 initial commit

### PR DESCRIPTION
**Pull request by fcabrales-plos**

### CODE REVIEW

File: test/api/RequestObject/ingestion_json.py [hunk 1]
```diff
@@ -212,10 +103,9 @@ def verify_ingestion_attribute(self, actual_results, expected_results, attribute
   def verify_array(self, actual_array, expected_array):
     self.assertIsNotNone(actual_array)
     self.assertIsNotNone(expected_array)
-    #self.assertEquals(len(actual_array), len(expected_array), 'The arrays have a different size')
 
   """
-  Below SQL statements will get article id from ambra db given doi  example 10.1371/journal.pone.0155391
+  Below SQL statements will query ambra article table given doi  example 10.1371/journal.pone.0155391 for article id
```
Comment (sbassi) [inline test/api/RequestObject/ingestion_json.py]: PEP-8 and other style guides recommends putting comments as docstrings, that is inside each function/method. State general purpose, parameters (name and data type and an exaplanation) and also information on what the methods returns. Will put only one note on this but it applies for all your changes.

Comment (fcabrales-plos) [inline test/api/RequestObject/ingestion_json.py]: Thanks, will let you know when changes are done. thanks @sbassi

**Comment by fcabrales-plos** [comment]
@sbassi  done with changes, please take a look.

File: test/api/RequestObject/ingestion_json.py [hunk 2]
```diff
@@ -95,103 +68,21 @@ def verify_journals(self):
...
-    """
-    print 'Verify Article\'s graphics'
-    self.verify_article_assets(article_id,'inline-formula, disp-formula', 'graphics')
+    article_id = self.get_article_id_sql_doi(resources.NOT_SCAPE_ARTICLE_DOI)
+    self.verify_article_assets(article_id, 'assetsLinkedFromManuscript')
 
-  def verify_article_assets(self, article_id, assets_type, assets_json_name):
-    assets = self.get_asset_figures_graphics(assets_type, article_id)
+  def verify_article_assets(self, article_id, assets_json_name):
+    assets = self.get_asset_figures_graphics(article_id)
```
Comment (sbassi) [inline test/api/RequestObject/ingestion_json.py]: parameters missing in the docstring

**Comment by fcabrales-plos** [comment]
@sbassi  Done with latest PR changes, please take a look.

**Pull request merged**